HotelBooking.Data/Repositories/AccountRepository.cs

```diff
@@ -34,83 +35,111 @@ public class AccountRepository : IAccountRepository
             this.tokenManager = tokenManager;
         }
 
-        public async Task<AuthenicationModel> ChangePassword(string email, ChangePasswordDTO model)
+        public async Task<ResponseModel> ChangePassword(string email, ChangePasswordRequest model)
         {
             var exist_user = await userManager.FindByEmailAsync(email);
```

Comment:
```suggestion
            var existUser = await userManager.FindByEmailAsync(email);
```
please fix other variables as well